src/fmu/dataio/providers/_fmu.py

```diff
@@ -179,6 +180,13 @@ def get_metadata(self) -> internal.FMUClassMetaData:
                 context=self._get_fmucontext_meta(),
                 model=self._get_fmumodel_meta() if self.model else case_meta.fmu.model,
                 workflow=self._get_workflow_meta() if self.workflow else None,
+                ert=fields.Ert(
+                    experiment=fields.Experiment(
+                        id=uuid.UUID(FmuEnv.EXPERIMENT_ID.value)
+                        if FmuEnv.EXPERIMENT_ID.value
+                        else None
+                    )
```

Comment:
Maybe we could pack this into a method `self._get_ert_meta()` since it is used two places? Then it will be more aligned with the rest 🙂

Reply:
Yes absolutely, good point!